src/styles/container.scss

```diff
@@ -0,0 +1,23 @@
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
You have set the column width to a fixed value of 200px. This could lead to issues with responsiveness, as the columns will not resize based on the viewport size. It's better to use fractional units (fr) for grid columns to make your layout more flexible.